Added footer

andreisavadev opened the pull request:
Hey,

I added the footer according to your specs in the open issue.

andreisavadev committed:
Added footer

Tiffufu commented:
@GowthamPB Please have a look at Andrei's pull request to see how he solved the problem. You'll see that he wrapped the footer HTML in the `<footer>` tags and then [applied this CSS here](https://github.com/Tiffufu/reading-calculator/blob/4387f794ec1d90c0976fbac55464c4a95f0a2f75/css/main.css#L165-L174). It looks like he finished this issue to spec, but it always helps to look at how other developers solve problems.

Tiffufu commented:
> Hey,
> 
> I added the footer according to your specs in the open issue.

@andreisavawebdev Andrei, you did a great job coding this to spec. It looks like the footer doesn't stick to the bottom of the page and there's a bunch of space underneath, but I understand that I didn't specify that the footer should be sticky.

Would you be able or willing to make the following changes?
- Change text color to use `#707070` for text color (I made a mistake with the purple!)
- Make it so that the footer sticks to the bottom of the page

If not, it's okay. I can always create another issue. Thanks for your help!

andreisavadev committed:
Changed footer color and made footer stick to bottom

andreisavadev committed:
Changed footer height

andreisavadev commented:
@Tiffufu hello. I changed the color and made the footer stick to the bottom. I also changed the footer height as it was too big at 200px and it messed with the responsiveness (the footer was hiding the content on smaller screens), cut it in half and it should be fine now.

GowthamPB commented:
@Tiffufu I'll surely look into the code. Thanks a lot :)
